tests/set-labels/wrong-config-group/fn-config.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: fn.kpt.dev/v1alpha1
+kind: SetLabel
```

Comment:
super nit: can we change the `SetLabel` to `BadKind`? the plural `SetLabels` is less obvious 🤣